tests/attention/test_dllm_vs_flex_attention.py

```
@@ -0,0 +1,1262 @@
+                t0 = time.perf_counter()
+                for _ in range(bench_iters):
+                    _run_fi()
+                torch.cuda.synchronize(device)
+                fi_ms = (time.perf_counter() - t0) / bench_iters * 1000
+
+                # cuda_graph latency
+                try:
+                    fi_cg_ms = benchmark_with_cuda_graph(_run_fi, warmup_iters, bench_iters)
+                    entry["fi_cg_ms"] = fi_cg_ms
+                except Exception:
+                    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)